docs/INSTALLATION.md

```diff
@@ -20,6 +20,7 @@ Build Server Requirements
 * **[email.php](https://github.com/fossasia/open-event-android/blob/master/apk-generator/api/email.sh)** A php script used to send the email containing the generated app as an attachment.
 * **[upload.php](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/upload.php)** A php script that will upload the json zip that is chosen by the user into a unique folder based on his timestamp.<br>  **[api.php](https://github.com/fossasia/open-event-android/blob/development/apk-generator/api/api.php)** A php script that serves as an API and handles incoming requests to the server.
 * **[copyapk.sh](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/copyApk.sh)** A bash script that will copy the generated and signed apk to the release folder in `public-html` of the server.
+* **[cleanup.sh](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/cleanup.sh)** A bash script that will delete all the signed apks from the release folder in `public-html` of the server. This is to be run as a cronjob using the command ``` crontab 01 00 * * * root Path_to_cleanup.sh ``` where 01 00 means it'll run at 1.00 AM every night and will execute cleanup.sh.
 

```

Comment:
@the-dagger This job runs at 1 AM and will iterate over all folders and delete the ones that were created 3 days back